backend-api/src/functions/asyncActiveSession/configService/configService.ts

```diff
@@ -14,7 +15,12 @@ export class ConfigService implements IGetConfig<Config> {
         errorCategory: "SERVER_ERROR",
       });
     }
-
+    if (!env.SESSION_TABLE_NAME) {
```

Comment:
feels like there's a bit of repetition in this file that we could avoid - not for now, but 🥘 for 🤔